pkg/lockfile/parse-pnpm-lock.go

```diff
@@ -23,6 +23,7 @@ type PnpmLockPackage struct {
 	Resolution PnpmLockPackageResolution `yaml:"resolution"`
 	Name       string                    `yaml:"name"`
 	Version    string                    `yaml:"version"`
+	Dev        string                    `yaml:"dev"`
```

Comment:
shouldn't we be able to parse this as a bool directly?

Reply:
good catch - I remember `bool` didn't work so I put `string` here but I just tried this again and `bool` is perfectly fine.